webextension/popup.html

```diff
@@ -106,25 +106,25 @@ <h3 class="panel-header-text">Edit Containers</h3>
       <div class="small-10 columns">
         <form>
           <p>
-            <label>Name</label>
+            <label for="edit-container-panel-name-input">Name</label>
             <input type="text" name="container-name" id="edit-container-panel-name-input"/>
           </p>
           <p>
-            <label>Choose a color</label>
+            <label for="edit-container-panel-choose-color">Choose a color</label><!-- TODO this needs moving to use radio --!>
```

Comment:
`<div>` is not a [labelable Form element](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Content_categories#Form_labelable).